packages/core/src/modules/dids/DidsApi.ts

```diff
@@ -1,37 +1,102 @@
-import type { Key } from '../../crypto'
-import type { DidResolutionOptions } from './types'
+import type {
+  DidCreateOptions,
+  DidCreateResult,
+  DidDeactivateOptions,
+  DidDeactivateResult,
+  DidResolutionOptions,
+  DidUpdateOptions,
+  DidUpdateResult,
+} from './types'
 
 import { AgentContext } from '../../agent'
 import { injectable } from '../../plugins'
 
+import { DidsModuleConfig } from './DidsModuleConfig'
 import { DidRepository } from './repository'
-import { DidResolverService } from './services/DidResolverService'
+import { DidRegistrarService, DidResolverService } from './services'
 
 @injectable()
 export class DidsApi {
+  public config: DidsModuleConfig
+
   private resolverService: DidResolverService
+  private registrarService: DidRegistrarService
   private didRepository: DidRepository
   private agentContext: AgentContext
 
-  public constructor(resolverService: DidResolverService, didRepository: DidRepository, agentContext: AgentContext) {
+  public constructor(
+    resolverService: DidResolverService,
+    registrarService: DidRegistrarService,
+    didRepository: DidRepository,
```

Comment:
hmm maybe. I think it is useful to have the prefix because we often deal with multiple repositories. The resolver/registrar is used less often so it's more distinctive on its own. I'll make it `didResolverService` and `didRegistrarService`